Source/Dafny/Compiler-java.cs

```diff
@@ -475,8 +475,7 @@ protected class ClassWriter : IClassWriter {
     }
 
     private void EmitSuppression(TextWriter wr) {
-      wr.WriteLine("@SuppressWarnings(\"unchecked\")");
-      wr.WriteLine("@SuppressWarnings(\"deprecation\")");
+      wr.WriteLine("@SuppressWarnings({\"unchecked\", \"deprecation\"})");
```

Comment:
Suppressing deprecation warnings might be an ok temporary fix, but did you find out which deprecated API you're using? The travis log:
```
+Note: ./_System/__default.java uses or overrides a deprecated API.
+Note: Recompile with -Xlint:deprecation for details.
+Note: /home/travis/build/dafny-lang/dafny/Test/comp/Extern/Library/MyInt.java uses or overrides a deprecated API.
+Note: Recompile with -Xlint:deprecation for details.
+Note: /home/travis/build/dafny-lang/dafny/Test/comp/Extern/_System/__default.java uses or overrides a deprecated API.
+Note: Recompile with -Xlint:deprecation for details.
```
suggests using `-Xlint:deprecation` to find that out, did you try that?